libtock/kernel/app_loader.h

```diff
@@ -0,0 +1,98 @@
+#pragma once
+
+#ifdef __cplusplus
+extern "C"
+{
+#endif
+
+#include "libtock/kernel/syscalls/app_loader_syscalls.h"
+#include "libtock/tock.h"
+
+#define BUTTON1 0
+#define BUTTON2 1
+#define BUTTON3 2
+
+/*
+ * Function to setup the callback from capsule.
+ * This function takes in the function that will be executed
+ * when the callback is triggered.
+ */
+returncode_t libtock_app_loader_set_setup_upcall(subscribe_upcall cb, void* userdata);
```

Comment:
These need to be updated to match https://github.com/tock/libtock-c/blob/3ff91f69e51ee305ab37e0878c88ba8a3a2b4ac2/doc/guide.md